packages/@groupby/elements-quickstart/test/unit/common/quick-start.test.ts

```diff
@@ -0,0 +1,168 @@
+    expect(SaytPlugin.calledWithNew()).to.be.true;
+    expect(SaytDriverPlugin.calledWithNew()).to.be.true;
+    expect(SearchPlugin.calledWithNew()).to.be.true;
+    expect(SearchDriverPlugin.calledWithNew()).to.be.true;
+  });
+
+  it('should register plugins', () => {
+    const cachePlugin = { cache: 'cache' };
+    const cacheDriverPlugin = { cacheDriver: 'cacheDriver' };
+    const domEventsPlugin = { domEvents: 'domEvents' };
+    const saytPlugin = { sayt: 'sayt' };
+    const saytDriverPlugin = { saytDriver: 'saytDriver' };
+    const searchPlugin = { search: 'search' };
+    const searchDriverPlugin = { searchDriver: 'searchDriver' };
+    CachePlugin.returns(cachePlugin);
+    CacheDriverPlugin.returns(cacheDriverPlugin);
+    DomEventsPlugin.returns(domEventsPlugin);
+    SaytPlugin.returns(saytPlugin);
+    SaytDriverPlugin.returns(saytDriverPlugin);
+    SearchPlugin.returns(searchPlugin);
+    SearchDriverPlugin.returns(searchDriverPlugin);
+
+    quickStart({ customerId });
+
+    const plugins = register.firstCall.args[0];
+    expect(plugins).to.have.members([
+      cacheDriverPlugin,
+      cachePlugin,
+      domEventsPlugin,
+      saytDriverPlugin,
+      saytPlugin,
+      searchDriverPlugin,
+      searchPlugin,
+    ]);
+  });
+
+  it('should forward customerId and options to the SaytPlugin', () => {
+    quickStart({ customerId, pluginOptions: { sayt: options } });
+
+    expect(SaytPlugin).to.be.calledWith({ ...options, subdomain: customerId });
+  });
+
+  it('should forward customerId and options to the SearchPlugin', () => {
+    quickStart({ customerId, pluginOptions: { search: options } });
+
+    expect(SearchPlugin).to.be.calledWith({ ...options, customerId });
+  });
+
+  it('should forward productTransformer to SaytDriverPlugin', () => {
+    quickStart({ customerId, productTransformer });
+
+    expect(SaytDriverPlugin).to.be.calledWith({ productTransformer });
+  });
+
+  it('should forward options to the SaytDriverPlugin', () => {
+    options.productTransformer = productTransformer;
+
+    quickStart({ customerId, pluginOptions: { sayt_driver: options } });
+
+    expect(SaytDriverPlugin).to.be.calledWith(options);
+  });
+
+  it('should pass the productTransformer from the sayt driver options if it exists instead of the general one', () => {
+    const optionsProductTransformer = stub();
+    options.productTransformer = optionsProductTransformer;
+
+    quickStart({ customerId, productTransformer, pluginOptions: { sayt_driver: options } });
+
+    expect(SaytDriverPlugin).to.be.calledWith(options);
+    expect(SaytDriverPlugin).to.not.be.calledWith(productTransformer);
+  });
+
+  it('should forward productTransformer to SearchDriverPlugin', () => {
+    quickStart({ customerId, productTransformer });
+
+    expect(SearchDriverPlugin).to.be.calledWith({ productTransformer });
+  });
+
+  it('should forward options to the SearchDriverPlugin', () => {
+    options.productTransformer = productTransformer;
```

Comment:
This line (or its level of specificity) may not be necessary for proving that `options` is indeed being passed.

Reply:
Good point. Will also remove this in the corresponding test for the SaytDriverPlugin.